cloudformation/airflow-ecs-services/worker.yaml

```diff
@@ -224,7 +224,7 @@ Resources:
             - Name: REDIS_DB_NUM
               Value: !Ref RedisDatabaseNumber
             - Name: GUNICORN_CMD_ARGS
-              Value: '--forwarded-allow-ips *'
+              Value: '--forwarded-allow-ips * --keep-alive 30'
             - Name: AIRFLOW__SMTP__SMTP_USER
               Value: !Ref SMTPUser
             - Name: SMTP_PASSWORD_ENC
```

Comment:
LGTM but what is the impact of increasing or decreasing the timeout, it would be valuable to document this

Reply:
http://docs.gunicorn.org/en/stable/settings.html#keepalive

![image](https://user-images.githubusercontent.com/19393946/46294058-f43e7180-c562-11e8-9bc8-16fbc258dac5.png)